### DISCUSSION TITLE
Errors when install dotnet

### DISCUSSION THREAD

**Discussion by mzxwt** [The-Wineskin-Project/WineskinServer#304] [General]
Errors occurred when install dotnet with engine WS11WineCX64Bit21.2.0 and Winetricks and it is failed to install dotnet.
<img width="372" alt="截屏2022-09-20 18 46 44" src="https://user-images.githubusercontent.com/26818282/191238926-32a19d3a-8334-4945-baca-e7adc1c9abfb.png">

**Comment by Nemental** [comment]
I can confirm. If you wait a few minutes before dismissing the message and then immediately closing winetricks, the dotnet installation still appears to work, but is not marked as installed.

**Comment by Gcenx** [comment]
I’ll assume it’s something related to the system language as it’s working just fine for me and many others.

**Comment by Nemental** [comment]
Thanks for this information. My MacBook Pro M1 system language is German. Do you have any idea what I can try to fix this problem?

**Reply by mzxwt** [reply]
Thanks for your reply. I'm trying to change the system language settings from Chinese to English.

**Reply by mzxwt** [reply]
<img width="1271" alt="截屏2022-09-27 10 11 24" src="https://user-images.githubusercontent.com/26818282/192415783-edbced54-51ee-41d4-b0dc-ad73309c117c.png">
The error codes marked in the Winetricks.log may be the key reasons for the error message.

**Reply by Nemental** [reply]
I'm pretty sure the problem is in "Schlüssel" - because it contains an umlaut.
Same line positions of my Winetricks.log looks like this:

```
------------------------------------------------------
Executing load_remove_mono internal
reg: Das System konnte den angegebenen Schlüssel oder Wert nicht finden
reg: Das System konnte den angegebenen Schlüssel oder Wert nicht finden
Executing rm -f /Users/mopo/Applications/Wineskin/KeePass_net.app/Contents/SharedSupport/prefix/dosdevices/c:/windows/system32/mscoree.dll
Executing rm -f /Users/mopo/Applications/Wineskin/KeePass_net.app/Contents/SharedSupport/prefix/dosdevices/c:/windows/syswow64/mscoree.dll
Executing w_do_call dotnet40
Executing mkdir -p /Users/mopo/Applications/Wineskin/KeePass_net.app/Contents/SharedSupport
------------------------------------------------------
```

```
------------------------------------------------------
Executing load_remove_mono internal
reg: Das System konnte den angegebenen Schlüssel oder Wert nicht finden
reg: Das System konnte den angegebenen Schlüssel oder Wert nicht finden
Executing rm -f /Users/mopo/Applications/Wineskin/KeePass_net.app/Contents/SharedSupport/prefix/dosdevices/c:/windows/system32/mscoree.dll
Executing rm -f /Users/mopo/Applications/Wineskin/KeePass_net.app/Contents/SharedSupport/prefix/dosdevices/c:/windows/syswow64/mscoree.dll
Executing w_do_call winxp
Executing mkdir -p /Users/mopo/Applications/Wineskin/KeePass_net.app/Contents/SharedSupport
------------------------------------------------------
```
@Gcenx Is there a way to handle the encoding in WineSkin?
